<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/lena.jpg</normaloff>:/images/lena.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topBrowseLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputImageTextInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseInputBtn">
        <property name="text">
         <string>Browse</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>lena.jpg</normaloff>lena.jpg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="filterType">
      <property name="title">
       <string>Filter Type</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="medianBlurBtn">
         <property name="text">
          <string>Median Blur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="GaussianBlurBtn">
         <property name="text">
          <string>Gaussian Blur</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomOutLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outImageInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseOutputBtn">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="displayImg">
      <property name="text">
       <string>Display Image after Saving</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
